include/arch/arm/cortex_m/scripts/linker.ld

```diff
@@ -172,6 +172,11 @@ SECTIONS
 	KEEP(*(".kinetis_flash_config.*"))
 
 	_vector_end = .;
+	/* This is used for Secure Boot */
+	#if defined(CONFIG_SB_FIRMWARE_INFO_OFFSET)
```

Comment:
Is it posible to not define CONFIG_SB_FIRMWARE_INFO_OFFSET while building using fw-nrfconnect-nrf?

Reply:
Where is the switch to turn this off?

Reply:
It's turned on of by `CONFIG_SECURE_BOOT` I guess setting `CONFIG_SECURE_BOOT` would be more sensible I'll change it. It was a copy pasta from old linker script.

Reply:
As for where it is set it's set here https://github.com/NordicPlayground/fw-nrfconnect-nrf/blob/9816a684f31d62aad02fb6f7ead41dbb9320282a/subsys/bootloader/fw_metadata/Kconfig#L17

which is included here https://github.com/NordicPlayground/fw-nrfconnect-nrf/blob/3a15a14d17dbe3129514a228b34729d00e23ee09/subsys/bootloader/Kconfig#L104

Reply:
Changed it so it should make more sense now @nvlsianpu